app/views/internal/users/_user_organizations.html.erb

```diff
@@ -14,27 +14,35 @@
         <%= f.submit "Add to Org", class: "btn btn-primary" %>
       </div>
     <% end %>
-    <% if @user.organization_memberships.size.positive? %>
+    <% if @organization_memberships.any? %>
```

Comment:
Nice one.

I think I'm going to change to `load.empty` approach, calling `load` on controller seems funky to me. 

Altho, if we call load in the controller it won't make check call. Since it's already loaded.

https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-empty-3F